giskard/visualization/templates/scan_report/markdown/huggingface.md

```diff
@@ -0,0 +1,38 @@
+Hey Team!🤗✨ 
+
+We’re thrilled to share some amazing evaluation results that’ll make your day!🎉📊
```

Comment:
❓Question: Do you think we should keep this part in the template or [the cicd posting script](https://github.com/Giskard-AI/cicd/pull/8)?